### ISSUE TITLE
Allow skipping indentation on the first line

### ISSUE THREAD

**Issue by Nemo157** [Lucretiel/indent-write#3]
I want to write some data to a stream, with a custom prefix, then each line under it indented. I can't simply write the first line and then swap out the writer with one wrapped in `IndentWriter` because the newlines are part of one of the items formatted to the stream.

As an example of what I'm trying to do, something like:

```rust
use std::io::Write;

fn main() {
    let foo = [1, 2];
    let bar = [4, 5];
    let mut out = indent_write::io::IndentWriter::new("     ", std::io::stdout()).skip_first();
    writeln!(out, "with foo: {:#?}\nbar: {:#?}", foo, bar).unwrap();
}
```

would output:

```
with foo: [
         1,
         2,
     ]
     bar: [
         4,
         5,
     ]
```

**Comment by Lucretiel** [comment]
Thanks for the suggestion! I'm adding a new constructor to `IndentWriter` to take care of this.

**Comment by Lucretiel** [comment]
Published in 2.1.0
